includes/scripts.php

```diff
@@ -15,8 +15,10 @@
  * @uses wp_enqueue_script()    Enqueues javascript.
  */
 function d7_enqueue_scripts() {
-	$script_location = 'js/main.js';
-	wp_enqueue_script('basic', get_bloginfo('template_directory') . '/' . $script_location , array('jquery'), '1.0');
```

Comment:
Yes, it's loading the copy of jQuery that is bundled with WP. The dependencies option of the wp_enqueue_script function triggers the bundled jQuery to be loaded.